.gitlab-ci.yml

```
@@ -327,6 +327,9 @@ update-nodejs-agent:
   stage: update-deps
   needs: []
   dependencies: []
+  before_script:
+    - !reference [.go-cache, before_script]
+    - apt-get update && apt-get install -y jq
```

```suggestion
    - apt update && apt install -y jq
```
Should we be using `apt` instead of `apt-get`? (Not sure if it makes much of a difference here, but we're using `apt` in the workflows already, not `apt-get`)